backend/audit/templates/audit/not-ready-for-certification.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<div class="grid-container margin-top-6">
+  <div class="grid-row grid-gap">
+      <form class="usa-form usa-form--large" id="ready-for-certification" method="post">
+          {% csrf_token %}
+          <fieldset class="usa-fieldset">
+              <legend class="usa-legend usa-legend--large" id="federal-awards">
+                  Not ready for Certification
+              </legend>
+              <p>While validating across sheets we encountered the errors below. You must correct these errors in order to proceed to certification.</p>
+              <p>Errors:</p>
+              <pre>{{ errors.errors | pprint}}</pre>
+              <p>Data submitted:</p>
+              <pre>{{ errors.data | pprint}}</pre>
+              <p>Audit {{ report_id }} has a submission status of: {{ submission_status }}</p>
+          </fieldset>
+      </form>
```

Comment:
**question**: Should this page contain an action button to link people back to the `report-submission` progress page so that they can start fixing these errors?